app/uat_actions/uat_actions/generate_tag_layout_template.rb

```diff
@@ -30,15 +30,15 @@ class UatActions::GenerateTagLayoutTemplate < UatActions
              :select,
              label: 'Direction',
              help: 'Direction the tags are laid out by',
-             select_options: -> { TagLayoutTemplatesController::DIRECTIONS },
+             select_options: -> { TagLayout::DIRECTIONS },
```

Comment:
`app/controllers/tag_layout_templates_controller.rb` has a similar constant but keys are different and it misses `TagLayout::CombByRows` . The controller makes it available to the view `app/views/tag_layout_templates/new.html.erb` as `@direction_algorithms` . To avoid duplication and for consistency, I think we should also
1. transfer the comments and the examples in the keys from the controller to the TagLayout for DIRECTIONS 
2. remove the constant from the controller
3. assign @direction_algorithms to TagLayout::DIRECTIONS in the controller

Reply:
Ah this needs more thought because I remember now that they were different lists on purpose. We didn't want to expose all of the options through the UAT action. Although I cannot remember why now. Will take another look, thanks for the comment.

Reply:
So we need a kind of 'display' version of the keys, which was the reason for the list in the controller. Agree it's not ideal to duplicate it though.

Reply:
Also, for some reason, the GUI for tag_layout_template/new doesn't allow you to set the walking by option, it fixes it to wells of plate. You can only set Direction. That seems inflexible.

The UAT action isn't using these descriptive keys, I was wrong there.